### ISSUE TITLE
Upgrade to Gradle 7

### ISSUE THREAD

**Issue by ohltyler** [opensearch-project/common-utils#112]
This component should be upgraded to gradle 7.0+ to unblock 1.3.0 distribution builds, which are currently broken. More details in tracking ticket in opensearch-build: https://github.com/opensearch-project/opensearch-build/issues/1587

**Comment by saratvemulapalli** [comment]
Duplicate of : https://github.com/opensearch-project/common-utils/issues/101 ?

**Comment by ohltyler** [comment]
Right, I guess because of https://github.com/opensearch-project/OpenSearch/pull/2033 this should be expedited for 1.3.0 rather than 2.0.0. Closing this one.
